rackcity/views/it_model_views.py

```diff
@@ -267,7 +267,8 @@ def records_are_identical(existing_data, new_data):
         if key not in new_keys and existing_data[key] is not None and key != 'id':
```

Comment:
I'm worried about these conditionals getting hard to track. Could you possibly add some parentheses between the `and`s so it's a little more readable?

Reply:
Ditto